searchcore/src/vespa/searchcore/proton/documentmetastore/i_store.h

```diff
@@ -6,6 +6,7 @@
 #include <vespa/document/base/globalid.h>
 #include <vespa/document/bucket/bucketid.h>
 #include <persistence/spi/types.h>
+#include <vespa/searchlib/common/serialnum.h>
```

Comment:
Do you need this when you are using uint64_t instead of SerialNum

Reply:
No. Removed.